README.md

```diff
@@ -2,11 +2,45 @@
 Allows for full control of the PicoBrew Pico S/C/Pro & Zymatic models.  Shout out to [@hotzenklotz](https://github.com/hotzenklotz/picobrew-server), Brian Moineau for PicoFerm API, @tmack8001 for Z series support & updates.  
 [Demo Server](http://ec2-3-136-112-93.us-east-2.compute.amazonaws.com/)
 
+## Supported Devices
+* Pico S/C/Pro: fully featured
+* Zymatic: fully featured
+* ZSeries (Beta, with additional development requirements for SSL support)
+  * Working (with limited testing)
+    * boot up sequence
+    * firmware updating / sideloading
+    * recipe summary
+    * session create
+    * session reporting
+    * close session
+    * resumable session
+* PicoFerm (Beta - Currently terminates fermentation after 14 days)
+
+## Features 
+* Device Aliasing
+* Brew Sessions
+  * Live Graphing
+  * Historical Graphing
+* Recipe Library
+  * View Previously Created
+  * Create New Recipes
+* Manual Recipe Editing
+  * **Note** The table for adding/removing/editing recipe steps has several validation checks in it, but there is always the possiblity of ruining your Pico.  
+  * *For Pico S/C/Pro Only*: DO NOT EDIT or MOVE Rows 1-3 (Preparing to Brew/Heating/Dough In).  Drain times should all be 0 except for Mash Out (2 minutes) and the last hop addition (5 minutes) (for example, if you only have Hops 1 & 2, set the drain time on Hops 2 to 5, and remove the Hops 3 and 4 rows)
+
+## Installation
+
+Refer to the [Releases Page](https://github.com/chiefwigms/picobrew_pico/releases) for steps to get up and running with your own Pico server. 
+
+The remainder of this guide oriented around creating a development environment 
+
+# Development Setup
+
 ## Requirements
 
 DNS Forwarding (either through a router, RaspberryPi etc)  
   - Have a Raspberry Pi Zero W : https://albeec13.github.io/2017/09/26/raspberry-pi-zero-w-simultaneous-ap-and-managed-mode-wifi/
```

Comment:
no longer needed

Reply:
I've been waiting to add a config.yaml change that identifies the "fork" (repo) and branch for retrieving updates from. This would allow for an easy development setup from the web UI just type in a fully resolve git source link and hit "update and restart"... Done fetches from repo and pulls/resets hard (potentially) to the development branch.

This would reduce the time needed for testing a new release or feature from another contributor and overall make it easier for us to test and load the changes we are making ourselves without needing to go into ssh all the time.

Reply:
Could simply change that dev setup to refer to the raspberrypi image created instead I think is what Pete is after saying it isn't needed anymore.

Reply:
right - i was referring to line 8 since we have the pi image

Reply:
Not quite sure what to do for this part. Possibly might make sense in a subsequent pr.

Reply:
sure let's solve dev documentation as a separate task sure